src/App.jsx

```diff
@@ -1,104 +1,13 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
```

Comment:
Ensure that the `postsFromServer` import path is correct and that the `posts.json` file exists in the `api` directory. This is crucial for the `PostList` component to receive the data it needs.